src/App/DataSource/Email/Email.php

```diff
@@ -21,7 +21,7 @@
 use Ushahidi\Core\Entity\Contact;
 use Log;
 
-class Email implements IncomingAPIDataSource, OutgoingAPIDataSource
+class Email extends OutgoingEmail implements IncomingAPIDataSource
```

Comment:
Still extends outgoing email, partly because I wasn't sure what exactly might break if we had to send and receive through different sources